### ISSUE TITLE
Include copyright info in LICENSE

### ISSUE THREAD

**Issue by nocarryr** [WalkerKnapp/devolay#28]
Hello, I'm building a project using your library (thanks for the work btw) and wish to include it in binary form.

In documenting licensing, I'd like to point to your LICENSE file, but the info on this line isn't filled in: https://github.com/WalkerKnapp/devolay/blob/254336497a3f48ef6b5b5a9c1ec637aa572c2446/LICENSE#L189

Not urgent, but I just thought you should be aware.  Thanks again!

**Comment by WalkerKnapp** [comment]
Hello! Thanks for your interest in the project.

If I am understanding the text of the license correctly, I believe that section is supposed to serve as a template for headers on specific files (i.e. https://github.com/tensorflow/tensorflow/blob/d314c2e738bffe292d0b09eb87e1d43d90b8996d/tensorflow/__init__.py#L1), which I felt were unnecessary for this project. I don't believe it's supposed to be filled in for the LICENSE file.

I know that a lot of projects use this header blurb as the copyright notice, but that doesn't seem right to me personally. As far as I can tell, the license wants users to include it verbatim. I would prefer if a binary included text like this in the licenses disclaimer:
```
*********************************************************************************
Devolay (https://github.com/WalkerKnapp/devolay)
Licensed under the Apache License 2.0

                                 Apache License
                           Version 2.0, January 2004
                        http://www.apache.org/licenses/

   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION

   1. Definitions.

      "License" shall mean the terms and conditions for use, reproduction,
      and distribution as defined by Sections 1 through 9 of this document.

< ... the remainder of the LICENSE file ... >
```

Or if space is limited, a reference to the distribution by Apache seems like a common way to acknowledge it.
```
Devolay (https://github.com/WalkerKnapp/devolay)
Licensed under the Apache License 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
```

Honestly, my binaries should probably include a copy of the license themselves so this becomes a non-issue. I'll try to resolve that in a future version.

Also, make sure you have read over https://github.com/WalkerKnapp/devolay#%EF%B8%8Flicensing-considerations%EF%B8%8F and are staying within NewTek's licensing requirements. Apache 2.0 is fairly permissive, and I'm not going to be causing any problems over the use of this project, but NewTek definitely cares more.

Let me know if I can help any more with this! I always love to hear about projects that find my code useful :)

**Comment by nocarryr** [comment]
Thanks for the suggestions and quick response!

I placed your license alongside the jar file in my repo and linked to it in the readme (as well as following the Newtek licensing info as much as possible).

https://github.com/nocarryr/ndiMultiview-pde

For now, I'm only including the dynamically linked version of the library until I'm sure I've got everything right with Newtek's requirements.

Thanks again
